sdk/communication/Azure.Communication.Calling.Server/src/CallClient.cs

```diff
@@ -0,0 +1,604 @@
+        public virtual Response DeleteCall(string callLegId, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallClient)}.{nameof(DeleteCall)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNullOrEmpty(callLegId, nameof(callLegId));
+
+                return RestClient.DeleteCall(callLegId, cancellationToken);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary> Disconnect the current caller in a Group-call or end a p2p-call asynchronously.</summary>
+        /// <param name="callLegId"> The call leg id. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="callLegId"/> is null.</exception>
+        public virtual async Task<Response> HangupCallAsync(string callLegId, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallClient)}.{nameof(HangupCall)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNullOrEmpty(callLegId, nameof(callLegId));
+                return await RestClient.HangupCallAsync(callLegId, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary> Disconnect the current caller in a Group-call or end a p2p-call. </summary>
+        /// <param name="callLegId"> The call leg id. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="callLegId"/> is null.</exception>
+        public virtual Response HangupCall(string callLegId, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallClient)}.{nameof(HangupCall)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNullOrEmpty(callLegId, nameof(callLegId));
+                return RestClient.HangupCall(callLegId, cancellationToken);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /*
+
+        /// <summary> Subscribe to tone.</summary>
```

Comment:
Why is this commented out? If we don't want to include this, lets clean it up?

Reply:
I've clean up these lines.